pkg/plugins/runtime/k8s/controllers/namespace_controller.go

```diff
@@ -27,17 +32,20 @@ type NamespaceReconciler struct {
 	Log logr.Logger
 
 	SystemNamespace     string
+	CNIEnabled          bool
 	ResourceManager     core_manager.ResourceManager
 	DefaultMeshTemplate mesh_proto.Mesh
 }
 
+// Reconcile is in charge for two things:
+//   - create NetworkAttachmentDefinition if CNI enabled and namespace has label 'kuma.io/sidecar-injection: enabled'
+//   - create 'default' mesh for the namespace 'SystemNamespace'
```

Comment:
Not entirely true, since Mesh is ClusterScope object (it was NamespaceScoped at some point and that's why we create this in NamespaceReconciller which is a little bit off.

Reply:
Is it okay to write just `create 'default' mesh for cluster`?

Reply:
yes